includes/class-scd-ext-drip-email.php

```diff
@@ -326,72 +325,91 @@ public function send_single_email_drip_notifications( $user_id, $lessons, $email
 		 * @param string $email_greeting Defaults to "Good Day $first_name"
 		 * @param int $user_id
 		 */
-		$email_body_text = Sensei_Content_Drip()->utils->check_for_translation(
+		$email_greeting = apply_filters( 'scd_email_greeting', __( 'Good Day', 'sensei-content-drip' ) . ' ' . $first_name );
+
+		// Get email body text.
+		$email_body = Sensei_Content_Drip()->utils->check_for_translation(
 			'The following lessons will become available today:',
 			'scd_email_body_notice_html'
 		);
 
-		$email_greeting     = sprintf( '<p>%s</p>', esc_html( apply_filters( 'scd_email_greeting', __( 'Good Day', 'sensei-content-drip' ) . ' ' . $first_name ) ) );
-		$email_body_notice  = '<p>' . esc_html( $email_body_text ) . '</p>';
-		$email_body_lessons = '';
-
-		// Get the footer from the settings and replace the shortcode [home_url] with the actual site url
-		$email_footer_text = Sensei_Content_Drip()->utils->check_for_translation(
+		// Get email footer text.
+		$footer_text = Sensei_Content_Drip()->utils->check_for_translation(
 			'Visit the online course today to start taking the lessons: [home_url]',
-			'scd_email_footer_html' );
+			'scd_email_footer_html'
+		);
+		$email_footer = str_ireplace(
+			'[home_url]' ,
+			'<a href="' . esc_attr( home_url() ) . '" >' . esc_html( home_url() ) . '</a>' ,
```

Comment:
`esc_attr` and `esc_html` should both be `esc_url` here.

Reply:
Good catch! Done - 87a2442